### PR TITLE
TSDoc for ExecutionContext and friends.

### PR DESCRIPTION
**Pull request by jonathan-codaio**
PTAL @ekoleda-codaio @coda/packs

### CODE REVIEW

File: api_types.ts [hunk 1]
```diff
@@ -351,40 +351,168 @@ export interface Network {
...
   status: number;
+  /**
+   * The body of the response.
+   *
+   * If the response contains JSON data, either because the Content-Type header is application/json
+   * or if the data is JSON-parseable, this will be a parsed JavaScript object.
+   * Similarly, if the response headers are text/xml or application/xml, this will be a parsed
+   * JavaScript object using the `xml2js` library.
+   *
+   * If implicit parsing is undesirable, you may use {@link isBinaryResponse} on the request
```
Comment (ekoleda-codaio) [inline api_types.ts:422]: Should you mention that it returns a Node.js Buffer in this case?

File: api_types.ts [hunk 2]
```diff
@@ -351,40 +351,168 @@ export interface Network {
...
+ * You will be returned a URL that you can then return with your formula response, and
+ * Coda will ingest the data from that URL into permanent storage.
+ */
 export interface TemporaryBlobStorage {
+  /**
+   * Fetches the data at the given URL, applying user authentication credentials as appropriate,
+   * and stores it in Coda-hosted temporary storage. Returns a URL for the temporary file
+   * that you should return in your formula response.
+   *
+   * The URL expires after 15 minutes by default, but you may pass a custom expiry, however
```
Comment (ekoleda-codaio) [inline api_types.ts:482]: Is it really 15 minutes? In my tests a few weeks back it seemed to be closer to 15 seconds.

File: api_types.ts (hunk 1 above)
Comment (jonathan-codaio) [reply api_types.ts:422]: Yep good point, updated.

File: api_types.ts (hunk 2 above)
Comment (jonathan-codaio) [reply api_types.ts:482]: As best I can tell it's 15 minutes: https://github.com/coda/coda/blob/main/modules/server/doc-bulk-storage/document_bulk_storage.ts#L567

**Review by ekoleda-codaio — APPROVED** [review]

**Pull request merged**